.travis.yml

```diff
@@ -53,17 +53,37 @@ jobs:
       name: bionic-arm64
       language: minimal
 
+  allow_failures:
+    - name: xenial-arm64
+
 env:
   global:
-  - VERSION=3.8-dev
-  # - ALIAS=nightly
+  - VERSION=3.9-dev
+  - ALIAS=nightly
```

Comment:
This needs to be removed (or left commented); `ALIAS` will be set by the API payload.

Reply:
Done.